circuit_maintenance_parser/parsers/hgc.py

```diff
@@ -0,0 +1,132 @@
+"""HGC parser."""
+import logging
+import re
+
+from dateutil import parser
+
+from circuit_maintenance_parser.errors import ParserError
+from circuit_maintenance_parser.parser import EmailSubjectParser, Html, Impact, CircuitImpact, Status
+
+# pylint: disable=too-many-branches
+
+
+logger = logging.getLogger(__name__)
+
+
+class SubjectParserHGC1(EmailSubjectParser):
+    """HGC subject parser."""
+
+    def parse_subject(self, subject):
+        """Parse HGC subject string.
+
+        Examples:
+            HGC Maintenance Work Notification - Network to Code _ CIR0000001 (TIC00000000000001)
+            HGC Maintenance Work Notification - Network to Code | CIR0000001 | TIC00000000000001
+        """
+        data = {}
+        try:
+            search = re.search(r"^.+\((.+)\)", subject.replace("\n", ""))
+            if search:
+                data["maintenance_id"] = search.group(1)
+            else:
+                split = subject.split(" | ")
+                data["maintenance_id"] = split[2]
+            return [data]
+        except Exception as exc:
+            raise ParserError from exc
```

Comment:
```suggestion
        search = re.search(r"^.+\((.+)\)", subject.replace("\n", ""))
        if search:
            data["maintenance_id"] = search.group(1)
        else:
            split = subject.split(" | ")
            data["maintenance_id"] = split[2]
        return [data]
```
the main `parse` method is taking care of this (merge from develop!)